nbgrader/apps/feedbackapp.py

```diff
@@ -37,38 +26,74 @@ class FeedbackApp(BaseNbConvertApp):
 
     examples = Unicode(dedent(
```

Comment:
Yep, you [are right])https://github.com/ipython/ipython/blob/master/IPython/config/application.py#L372-L383) if might not need to be a traitlet, nor be be detended IIUC.